src/main/java/pro/belbix/ethparser/web3/uniswap/parser/UniswapLpLogParser.java

```diff
@@ -117,6 +117,10 @@ public UniswapDTO parse(Log ethLog, String network) {
 
     //enrich owner
     TransactionReceipt receipt = web3Functions.fetchTransactionReceipt(dto.getHash(), ETH_NETWORK);
+    if (receipt == null) {
```

Comment:
add checking status, if status=0 it means transaction failed
at least print this info to the log